src/url-utils.ts

```diff
@@ -1,4 +1,12 @@
-import { resolve } from 'url';
+export function resolve(from: string, to: string) {
```

Comment:
What about `resolveUrl` instead for a name? When it's in the context of `url-utils`, it's easy to know what this does but once its imported and used in another file, it becomes hard to know what this does from the name without looking at its import location

Reply:
yes. this was a case of me just coping the old deprecated method from url. Agree that resolveUrl is a better name. fixed!